src/services/UserService.ts

```diff
@@ -98,6 +98,22 @@ export class UserService {
 		});
 	}
 
+	public async resendEmail(email: string): Promise<APIPrivateUser> {
```

Comment:
Can you rename this to something like `getUserByEmail`? Since it doesn't actually send an email itself.